### DISCUSSION TITLE
How to make the chat assistant's final response be split into multiple replies like a real person？

### DISCUSSION THREAD

**Discussion by nlp999** [langgenius/dify#10236] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I would like to inquire about how to make the chat assistant's final response be split into multiple replies, as it is currently delivering quite long responses all at once.For example, I want the chat assistant to respond with one or two short sentences at a time, just like a real person, to make it appear more human-like.

**Comment by fsea** [comment]
I'd also like to know how such a feature should be implemented
